### PR TITLE
fix: use merged widget state in widget-form store and set first variable item correctly in DashboardVariablesDropdown

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

Please review in order by commits.

### CODE REVIEW

File: apps/web/src/services/dashboards/shared/dashboard-widget-input-form/helpers/form-data-helper.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { cloneDeep } from 'lodash';
+
+import type { DashboardVariablesSchema } from '@/services/dashboards/config';
+import type { PartialDashboardLayoutWidgetInfo } from '@/services/dashboards/store/widget-form';
+import type { InheritOptions, WidgetOptions } from '@/services/dashboards/widgets/_configs/config';
+
+
+export const getInitialFormData = (widgetInfo: PartialDashboardLayoutWidgetInfo, variableSchema: DashboardVariablesSchema): Record<string, any> => {
```
Comment (piggggggggy) [inline apps/web/src/services/dashboards/shared/dashboard-widget-input-form/helpers/form-data-helper.ts:8]: question: Is this separation for reuse?

**Review by piggggggggy — APPROVED** [review]
LGTM!!! Thank you 🥲

File: apps/web/src/services/dashboards/shared/dashboard-widget-input-form/helpers/form-data-helper.ts (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/dashboards/shared/dashboard-widget-input-form/helpers/form-data-helper.ts:8]: No, this is just for readability.

**Review by sulmoJ — APPROVED** [review]
I tried to review it as thoroughly as possible, but I found it difficult to understand exactly. It seems like we may need approval from @yuda110.

LGTM

**Review by yuda110 — APPROVED** [review]
LGTM!!!

**Pull request merged**